test/cache_test.rb

```diff
@@ -452,6 +460,9 @@ def test_fragment_fetch_with_namespaced_object
         id: @spam.id
       }
       assert_equal(@spam_hash, expected_result)
+    ensure
+      fragmented_serializer = @spam_serializer
+      Object.send(:remove_const, fragmented_serializer._get_or_create_fragment_cached_serializer.name)
```

Comment:
related to https://github.com/rails-api/active_model_serializers/pull/1781/files#r66014031